skipchain/skipchain.go

```diff
@@ -536,19 +536,18 @@ func (s *Service) GetSingleBlockByIndex(id *GetSingleBlockByIndex) (*SkipBlock,
 	return nil, errors.New("No block with this index found")
 }
 
-// GetAllSkipchains returns a list of all known skipchains
-func (s *Service) GetAllSkipchains(id *GetAllSkipchains) (*GetAllSkipchainsReply, error) {
-	// Write all known skipblocks to a map, thus removing double blocks.
-	chains, err := s.db.getAll()
+// GetAllSkipchains returns the latest block of all known skipchains.
+func (s *Service) GetAllSkipchains(_ *GetAllSkipchains) (*GetAllSkipchainsReply, error) {
```

Comment:
Please chose one of:
1. keep the old method and add a new one (like I did with `AddForwardLink` in #1267)
2. break it completely by changing the signature, so we know where it breaks and who else is using it

I have a strong preference for 1. and changing it in September for the upcoming `onet.v3`